lib/BridgedRoom.js

```diff
@@ -389,6 +446,68 @@ BridgedRoom.prototype.onSlackMessage = async function(message) {
     }
 };
 
+BridgedRoom.prototype.onSlackReactionAdded = async function(message) {
+    if (message.user_id === this._slack_user_id) {
+        return;
+    }
+    const ghost = await this._main.getGhostForSlackMessage(message);
+    await ghost.update(message, this);
+
+    const reaction = `:${message.reaction}:`;
+    let reaction_key = emoji.emojify(reaction, substitutions.onMissingEmoji);
+
+    const eventStore = this._main.getEventStore();
+    const event = await eventStore.getEntryByRemoteId(message.item.channel, message.item.ts);
+
+    return ghost.sendReaction(this.getMatrixRoomId(), event.eventId, reaction_key,
+                              message.item.channel, message.event_ts);
+};
+
+BridgedRoom.prototype.getMatrixReactionsBySenderByKey = async function(eventId, sender, key) {
+    // The objective of this function is to hit the new
+    // rooms/roomId/aggregations endpoint for all the m.reaction events with the
+    // specified key.
+    // Then we walk that list and find the one with the sender we care about, and return that event.
+
+    // TODO: Not sure this works, having trouble testing it against a sufficiently new synapse.
+    return;
```

Comment:
Please just remove the func if it doesn't work. I'm not sure it's worth bloating the code base out with future things if the spec isn't even merged yet.